scripts/bash/run_conformance.sh

```diff
@@ -18,7 +18,7 @@ SRC_DIR=`dirname "$0"`
 source ${SRC_DIR}/enceladus_env.sh
 
 export CLASS=${CONF_CLASS}
-export JAR=${SPARK_JOBS_JAR}
+export JAR=${SPARK_JOBS_JAR_OVERRIDE:-$SPARK_JOBS_JAR}
```

Comment:
I would move this to `run_enceladus.sh` so it's actually present in the code only once.

The separation is due to historical reasons, when **Standardization** and **Conformance** were in different jar files.